gremlin-go/driver/serializer.go

```diff
@@ -320,5 +321,12 @@ func initDeserializers() {
 		// Metrics
 		metricsType:          metricsReader,
 		traversalMetricsType: traversalMetricsReader,
+
+		// Customer
+		customType: customTypeReader,
+	}
+
+	customDeserializers = map[string]reader{
```

Comment:
We could an option to provide this map during initialization. Would it make sense to add a property to the `DriverRemoteConnectionSettings` struct?

Reply:
and to `ClientSettings`

Reply:
or alternatively, you may add something like `CustomTypeSerializerRegistry` to store serializers for all custom types